profiler/options.go

```diff
@@ -71,6 +74,19 @@ func urlForSite(site string) (string, error) {
 	return u, err
 }
 
+// Check for API key constraints:
+func apiKeyCheck(key string) bool {
+	if optionAPIKeyLength != len(key) {
+		return false
+	}
+	for _, c := range key {
+		if !((c >= 'a' && c <= 'z') || (c >= '0' && c <= '9')) {
```

Comment:
You could also try:
```suggestion
		if !unicode.IsLetter(c) && !unicode.IsNumber(c) {
```

Reply:
Good call.  Looks like unicode.IsLetter and friends use the bitmasking technique from libc, so this is more readable and not slower.

Took your suggestion in ef1b11d, but added `!unicode.isLower(c)` as well.